### DISCUSSION TITLE
Model summary as a figure

### DISCUSSION THREAD

**Discussion by robjhyndman** [quarto-dev/quarto-cli#5592] [Q&A]
I'd like to include the output from a model summary (e.g., summary() applied to an lm object) as a figure in a quarto document. The following shows the output, but not within a figure environment or with a caption.

````
```{r}
#| label: fig-model
#| fig-cap: Fitted model
summary(lmobject)
```
````

**Comment by mcanouil** [comment]
You can't `summary()` produces text as output.
So you have to use external software to convert text into image.

PS: this is unrelated to Quarto.

---

if you are interested in cross-referencing (<https://quarto.org/docs/authoring/cross-references.html#code-listings>), below is a workaround for #1580 .

````qmd
---
title: "Quarto Playground"
format: html
---

This is a playground for Quarto.

```{r}
#| attr-source: "id=lst-model lst-cap='Fitted model'"
1 + 1
```


See @lst-model.
````

**Reply by cscheid** [reply]
Let me clarify: you can't _currently_ do that in 1.3.

In 1.4, we're going to change the way cross-referenceable "artifacts" work, so that if you tell Quarto that something is a figure (or a table, etc), Quarto will believe you and go along, irrespective of the content.
